system-x/services/ldap/src/main/java/software/tnb/ldap/service/Ldap.java

```diff
@@ -0,0 +1,28 @@
+package software.tnb.ldap.service;
+
+import software.tnb.common.deployment.WithDockerImage;
+import software.tnb.common.service.Service;
+import software.tnb.ldap.account.LdapAccount;
+import software.tnb.ldap.validation.LdapValidation;
+
+import com.unboundid.ldap.sdk.LDAPConnection;
+
+public abstract class Ldap extends Service<LdapAccount, LDAPConnection, LdapValidation> implements WithDockerImage {
+
+    public abstract String getUrl();
```

Comment:
rename to `url()` to match other methods style

Reply:
Done